### DISCUSSION TITLE
怎么对表格（非图片）做信息抽取？

### DISCUSSION THREAD

**Discussion by wencan** [PaddlePaddle/PaddleNLP#7365] [Q&A]
rt
表格，比如word中的表格，文字pdf中的表格，不是图片化的表格，不需要ocr。
目前我找到的案例，都是针对表格图片，做基于ocr的信息抽取

**Comment by echo0218** [comment]
要么根据文字及坐标，自己组装、复原表格；要么当成图片，OCR...不过处理效果都不完美
